src/components/ReplyCard/InReplyTo/InReplyToAudio.vue

```diff
@@ -23,16 +23,39 @@ export default {
     translations: {
       type: Object,
       default: () => ({})
+    },
+    document: {
+      type: Object
     }
   },
   data: () => ({
-    timeAudio: null
+    timeAudio: null,
+    audio: undefined
   }),
-  mounted() {
-    const audioElement = this.$el.querySelector('#audio')
+  mounted: async function() {
+    const me = this
```

Comment:
It is a question and not a suggestion: Why using the variable `me` and not `this`?

Reply:
It's a common practice in the Vue.js community